src/internal/m365/collection/site/restore.go

```diff
@@ -211,7 +211,7 @@ func RestoreListCollection(
 				restoreContainerName,
 				errs)
 			if err != nil &&
-				errors.Is(err, api.ErrCannotCreateWebTemplateExtension) {
+				errors.Is(err, api.ErrCannotCreateNonRestorableListTemplate) {
```

Comment:
* We don't need `err != nil` here since the following `errors.Is` should be sufficient.
* Can we add a `logger.CtxErr(ctx,err).Info` here?